code/src/pages/MovieDetails.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from "react"
+import { useParams } from "react-router-dom"
+import "./moviedetails.css"
+
+// In useState initial state is set to 2 object rendered on first mount while page is loading
+export const MovieDetails = () => {
+  const api_key = "bf1c38c5c54137f389c5c7971de1fe96"
+  const { movieId } = useParams()
+  const [movieDetails, setMovieDetails] = useState({ title: "loading...", overview: "loading..." })
+
+  useEffect(() => {
+    fetch(`https://api.themoviedb.org/3/movie/${movieId}?api_key=${api_key}&language=en-US`)
```

Comment:
Sometimes you will see 
`const BASE_URL = 'https://api.themoviedb.org/3'`

And then something like 
```javascript
fetch(`${BASE_URL}/movie/${movieId}?api_key=${api_key}&language=en-US`)
```